### PR TITLE
Merge 0.8

### PR DESCRIPTION
**Pull request by glennjacobs**
Merges changes from the `0.8` branch into `1.x`.

### CODE REVIEW

**Comment by glennjacobs** [comment]
@alecritson do you think we should have the `.github` folders on each package?

File: packages/admin/src/Models/Staff.php [hunk 1]
```diff
@@ -117,6 +124,16 @@ public function getFullNameAttribute(): string
...
     }
 
+    /**
+     * Get staff member's Gravatar URLs.
+     */
+    public function getGravatarAttribute(): string
+    {
+        $hash = md5(strtolower(trim($this->attributes['email'])));
+
+        return "https://www.gravatar.com/avatar/$hash?d=mp";
```
Comment (alecritson) [inline packages/admin/src/Models/Staff.php]: Through Lunar we usually do `"{$hash}"` when using string interpolation

**Review by alecritson — CHANGES_REQUESTED** [review]

File: packages/admin/src/Models/Staff.php (hunk 1 above)
Comment (glennjacobs) [reply packages/admin/src/Models/Staff.php]: This was just a merge, but i can update it.

**Comment by alecritson** [comment]
> @alecritson do you think we should have the `.github` folders on each package?

I assume we need them because they have specific workflows which auto close PRs?

**Comment by glennjacobs** [comment]
> > @alecritson do you think we should have the `.github` folders on each package?
> 
> I assume we need them because they have specific workflows which auto close PRs?

Yeah, they're to auto-close PRs on the read-only repos. It could be argued that all the packages should have them.

**Review by alecritson — DISMISSED** [review]

File: tests/core/Unit/Search/ProductOptionIndexerTest.php [hunk 2]
```diff
@@ -11,6 +11,6 @@
 
     $data = app(ProductOptionIndexer::class)->toSearchableArray($productOption);
 
-    expect($data['name_en'])->toEqual($productOption->name['en']);
-    expect($data['label_en'])->toEqual($productOption->label['en']);
+    $this->assertEquals($productOption->name->en, $data['name_en']);
```
Comment (alecritson) [inline tests/core/Unit/Search/ProductOptionIndexerTest.php]: Just realised after I approved that this has been reverted to PHPUnit syntax.

**Review by alecritson — APPROVED** [review]

**Pull request merged**